scanapi/tree/request_node.py

```diff
@@ -115,6 +115,10 @@ def retry(self):
         return self.spec.get(RETRY_KEY)
 
     def run(self):
+        """Make HTTP requests and generating test results for the given URLs.
+        Returns:
+            [dict]: HTTP response and test results with request node name, to be used by the report template.
```

Comment:
```suggestion
            [dict]: HTTP response and test results with request node name, 
            to be used by the report template.
```

Suggestion to fix [Doc line too long style error](https://deepsource.io/gh/scanapi/scanapi/run/22383366-111c-42d6-bd32-31240fce70f4/python/FLK-W505)